### DISCUSSION TITLE
Transcriptions : utilisation de la balise <orig> ?

### DISCUSSION THREAD

**Discussion by saranadal** [anrExperts/adire#25] [Q&A] [answered]
@emchateau tu proposais l'utilisation de la balise <orig> qui va souvent de paire avec la balise <reg>. Utilisée seule, je n'en vois pas trop l'utilité, mais peut-être que je me trompe.

**Comment by emchateau** [comment, marked as answer]
Je proposais d’utiliser `<orig>` lorsque tu avais explicitement besoin de marquer que ce n’était pas une faute de ta part. Mais comme le principe c’est de transcrire à l’identique, ce n’est généralement pas nécessaire
[ps pour mettre des balises dans du md il faut les entourer par ``  comme c’est du code]

**Reply by saranadal** [reply]
Ah oui c'est vrai ! Au temps pour moi.
[J'avais oublié qu'il fallait mettre des ``autour de code, j'ai fait attention ensuite !]
